### ISSUE TITLE
v3.1.0-alpha.35 TimePickerbody

### ISSUE THREAD

**Issue by mcuiobroker** [Zefau/ioBroker.jarvis#1841]
Bei Auswahl der Jetzt Zeit mit dem Jetzt-Button, wird automatisch das Popup aufgerufen.
![image](https://user-images.githubusercontent.com/84818934/187767610-9f009eab-4904-4bf3-bbcd-34f639f98b05.png)

**Comment by mcuiobroker** [comment]
Alternative?
![image](https://user-images.githubusercontent.com/84818934/187775480-54af8cec-67b2-4f0d-bf02-4b2ff0109292.png)
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/time

**Comment by Zefau** [comment]
Kann ich gerne übernehmen. Aber in dem Dropdown scheint immer nur eine limitierte Auswahl zur Verfügung zu stehen? Und die Anzeige ist zwischen Chrome und Firefox unterschiedlich, insbesondere bzgl. des Dropdowns selbst, welches es in Firefox nicht gibt.

**Comment by mcuiobroker** [comment]
Obwohl als kompatibel gesetzt
![image](https://user-images.githubusercontent.com/84818934/188287567-1287dca2-cb23-4d6f-9fd9-722727de3145.png)

**Comment by Zefau** [comment]
Jo, aber schau dir mal deinen Link an: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/time#appearance

**Comment by mcuiobroker** [comment]
Ja stimmt, würde es als Alternative Version (Umschaltbar in den Komponten Optionen) nutzen. Dort kann man dann schnell die Zahlen so eingeben. Wichtig dabei da der eingebene Wert dann auch gesetzt wird (ack: false / true).
